### ISSUE TITLE
The recommended command line dependencies install can break a user's system.

### ISSUE THREAD

**Issue by brunoais** [casualsnek/waydroid_script#107]
In:
```
git clone https://github.com/casualsnek/waydroid_script
cd waydroid_script
sudo python3 -m pip install -r requirements.txt
```

Instead of:
```
sudo python3 -m pip install -r requirements.txt
```
Use `--user`
```
sudo python3 -m pip install --user -r requirements.txt
```

That way these packages are not changed inside `/usr` which can break a user's system.
Even if a package still breaks a user's system, these installed packages can easily be removed with:
```
sudo python3 -m pip uninstall --user -r requirements.txt
```

That way the OS as a whole won't break (even if another software breaks but then it's just a matter of reinstalling the correct dependencies)

**Comment by amogus07** [comment]
Just install the dependencies without sudo as per the instructions and use a venv.

**Comment by brunoais** [comment]
Fixed the recommended commands. The sudo shouldn't have stayed in the copy&paste ups.

**Comment by brunoais** [comment]
Fixed in  #118
